src/lib.rs

```diff
@@ -0,0 +1,40 @@
+mod macros;
+mod types;
+
+use redis::{Client, RedisResult, Script};
+use std::fs::File;
+use std::io::prelude::*;
+use std::{thread, time};
+use types::BouncerStats;
+mod settings;
+use settings::Settings;
+pub fn run(key: u64, rate_limit: u8, wait_time: u8, b: &dyn Fn()) -> RedisResult<BouncerStats> {
+    match Settings::new() {
+        Ok(settings) => {
+            let mut file = File::open(settings.redis_script())?;
+            let mut script = String::new();
+            file.read_to_string(&mut script)?;
```

Comment:
std has a helper for this operation:

```suggestion
            let mut script = std::fs::read_to_string(settings.redis_script())?;
```